assertk-common/src/main/kotlin/assertk/assertions/support/support.kt

```diff
@@ -11,21 +11,9 @@ import assertk.fail
  */
 fun show(value: Any?, wrap: String = "<>"): String = "${wrap[0]}${display(value)}${wrap[1]}"
 
-private fun display(value: Any?): String {
-    return when (value) {
-        null -> "null"
-        is String -> "\"$value\""
-        is Class<*> -> value.name
-        is Array<*> -> value.joinToString(prefix = "[", postfix = "]", transform = ::display)
-        is Collection<*> -> value.joinToString(prefix = "[", postfix = "]", transform = ::display)
-        is Map<*, *> -> value.entries.joinToString(
-                prefix = "{",
-                postfix = "}",
-                transform = { (k, v) -> "${display(k)}=${display(v)}" })
-        is Regex -> "/$value/"
-        else -> value.toString()
-    }
-}
+expect
+@Suppress("UndocumentedPublicFunction")
```

Comment:
This strange syntax is needed to make `detekt` happy. But I am not happy with it :(
First, there is the split between expect and the rest of the signature that is non sense (detekt parsing problem ?).
Secondly, the method is internal and not public, I don't understand why we have the warning in the first place !

Reply:
Yeah noticed that on Travis. It really shouldn't need a doc since it isn't public. Maybe worth filling a detekt issue about it?

Reply:
Yes, definitely worth it !
Will do once I reproduced it with simpler code to give them :)